erpnext/accounts/doctype/payment_request/payment_request.py

```diff
@@ -291,31 +296,37 @@ def create_payment_entry(self, submit=True):
 
 		party_account_currency = ref_doc.get("party_account_currency") or get_account_currency(party_account)
 
-		bank_amount = self.grand_total
+		bank_amount = self.outstanding_amount
+
 		if party_account_currency == ref_doc.company_currency and party_account_currency != self.currency:
-			party_amount = ref_doc.get("base_rounded_total") or ref_doc.get("base_grand_total")
+			total = ref_doc.get("rounded_total") or ref_doc.get("grand_total")
```

Comment:
Set `party_amount` as `grand_total`, because PR is now created from party's bank account currency instead of transaction currency (reference doc currency).